### PR TITLE
Fix some documentation typos and errors

### PR DESCRIPTION
**Pull request by NobodysNightmare**
Found while working on https://github.com/opf/openproject/pull/3338

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://nobodysnightmare-feature-fix-doc.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://nobodysnightmare-feature-fix-doc.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: doc/apiv3-documentation.apib [hunk 1]
```diff
@@ -2060,7 +2060,7 @@ Revisions are sets of updates to files in the context of repositories linked in
 ## Local Properties
 | Property                | Description                                                                                                                                                 | Type        | Constraints | Supported operations |
 |:-----------------------:| ------------------------------------------------------------------------------------------------------------------------------------------------------------| ----------- | ----------- | -------------------- |
-| id                      | Revisions's id, assigned by OpenProject                                                                                                                     | Integer     | x > 0       | READ                 |
+| id                      | Revision's id, assigned by OpenProject                                                                                                                     | Integer     | x > 0       | READ                 |
```
Comment (oliverguenther) [inline doc/apiv3-documentation.apib]: Is that only in the GitHub file browser or is this line missing a space?

Comment (NobodysNightmare) [inline doc/apiv3-documentation.apib]: Yeah, that is totally GitHub messing things up... just hit F5 (aka _the reload button_) and the problem will be gone...

Comment (oliverguenther) [inline doc/apiv3-documentation.apib]: :see_no_evil:

**Pull request merged**